packages/pigeon/example/app/ios/Runner/Messages.g.swift

```diff
@@ -131,9 +131,12 @@ class ExampleHostApiSetup {
   /// The codec used by ExampleHostApi.
   static var codec: FlutterStandardMessageCodec { ExampleHostApiCodec.shared }
   /// Sets up an instance of `ExampleHostApi` to handle messages through the `binaryMessenger`.
-  static func setUp(binaryMessenger: FlutterBinaryMessenger, api: ExampleHostApi?) {
+  static func setUp(
+    binaryMessenger: FlutterBinaryMessenger, api: ExampleHostApi?, messageChannelSuffix: String = ""
+  ) {
     let getHostLanguageChannel = FlutterBasicMessageChannel(
-      name: "dev.flutter.pigeon.pigeon_example_package.ExampleHostApi.getHostLanguage",
+      name:
+        "dev.flutter.pigeon.pigeon_example_package.ExampleHostApi.getHostLanguage\(messageChannelSuffix.count > 0 ? ".\(messageChannelSuffix)" : "")",
```

Comment:
this can be just `"dev.flutter.pigeon.pigeon_example_package.ExampleHostApi.getHostLanguage\(messageChannelSuffix)"`

Reply:
Needs the `.` when it exists